exporter/views.py

```diff
@@ -14,7 +16,7 @@
 from concordia.models import Asset, Transcription
 
 
-def get_latest_transcription_data(campaign_slug):
+def get_latest_transcription_data(campaign_slug, filter_status=False):
```

Comment:
The name of this argument doesn't really tell me what it does and it seems likely that at some point we might want to have other filters. What do you think about changing this signature to be the asset queryset:

```suggestion
def get_latest_transcription_data(campaign_slug, asset_qs=None):
```

Then we could change the `Asset.objects` call below to use `asset_qs` and the caller could pass in an asset QuerySet with arbitrary filtering, ordering, select/prefetch related, etc.